### ISSUE TITLE
ikos-analyzer fails: error: llvm::Type and ar::Type alloc size are different

### ISSUE THREAD

**Issue by yurivict** [NASA-SW-VnV/ikos#37]
```
$ ikos test.cpp 
[*] Compiling test.cpp
[*] Running ikos preprocessor
[*] Running ikos analyzer
[*] Translating LLVM bitcode to AR
ikos-analyzer: /tmp/ikos-ON9hAd/test.pp.bc: error: llvm::Type and ar::Type alloc size are different
ikos: error: a run-time error occured
```

test.cpp is a testcase for the ```osg``` library (http://www.openscenegraph.org/).

[test.cpp.txt](https://github.com/NASA-SW-VnV/ikos/files/2681882/test.cpp.txt)

FreeBSD 11.2 amd64
ikos-2.1 installed from the port.
llvm-70, clang-70.

**Comment by practicalswift** [comment]
I've hit this issue as well.

Here is how to reproduce:

```
$ cat testcase.c
union {
  char a[0];
  int b
} c = {};
$ ikos testcase.c
[*] Compiling testcase.c
…
[*] Running ikos preprocessor
[*] Running ikos analyzer
[*] Translating LLVM bitcode to AR
ikos-analyzer: /tmp/ikos-WwsDgL/testcase.pp.bc: error: llvm::Type and ar::Type alloc size are different
ikos: error: a run-time error occurred
```

**Comment by arthaud** [comment]
Thank you. See the related issue #67

**Comment by arthaud** [comment]
This should be fixed by 1c12398390fdaba4797ea384ec9ec1172c2d9c7b. Feel free to reopen if the issue is still here.

**Comment by practicalswift** [comment]
Confirming that I'm not longer encountering this issue. Thanks!
